packages/oidc-middleware/README.md

```diff
@@ -139,10 +143,11 @@ Required config:
 * **issuer** - The OIDC provider (e.g. `https://{yourOktaDomain}/oauth2/default`)
 * **client_id** - An id provided when you create an OIDC app in your Okta Org
```

Comment:
Let's leave this for the future, but probably at some point we should standardize to all camelCase or all underscore_case.

Reply:
I think this is a transition in process